### ISSUE TITLE
gitpod support

### ISSUE THREAD

**Issue by akosyakov** [Genuitec/CodeTogether#11]
I am trying to install VS Code extension in Gitpod: https://www.gitpod.io/docs/vscode-extensions/#installing-an-extension

But running `Start Session` does nothing. Have you tried to run VS Code extension in Theia: https://github.com/eclipse-theia/theia/wiki/Testing-VS-Code-Extensions?

**Comment by akosyakov** [comment]
From server logs:
```
root ERROR [hosted-plugin: 8693] Promise rejection not handled in one second: Error: System keychain unavailable , reason: Error: System keychain unavailable
root ERROR [hosted-plugin: 8693] With stack trace: Error: System keychain unavailable
    at Object.<anonymous> (/tmp/vscode-unpacked/genuitecllc.codetogether-1.0.5.vsix/extension/out/extension.js:31:30994)
    at /tmp/vscode-unpacked/genuitecllc.codetogether-1.0.5.vsix/extension/out/extension.js:31:30568
    at Object.next (/tmp/vscode-unpacked/genuitecllc.codetogether-1.0.5.vsix/extension/out/extension.js:31:30673)
    at /tmp/vscode-unpacked/genuitecllc.codetogether-1.0.5.vsix/extension/out/extension.js:31:29575
    at new Promise (<anonymous>)
    at __awaiter (/tmp/vscode-unpacked/genuitecllc.codetogether-1.0.5.vsix/extension/out/extension.js:31:29320)
    at Object.getPassword (/tmp/vscode-unpacked/genuitecllc.codetogether-1.0.5.vsix/extension/out/extension.js:31:30910)
    at Object.<anonymous> (/tmp/vscode-unpacked/genuitecllc.codetogether-1.0.5.vsix/extension/out/extension.js:31:31839)
    at /tmp/vscode-unpacked/genuitecllc.codetogether-1.0.5.vsix/extension/out/extension.js:31:30568
    at Object.next (/tmp/vscode-unpacked/genuitecllc.codetogether-1.0.5.vsix/extension/out/extension.js:31:30673)
```

**Comment by akosyakov** [comment]
Is it about `keytar`? i.e. https://github.com/eclipse-theia/theia/issues/6407?

**Comment by TimWebb** [comment]
Yes, that is indeed the case. Have you stumbled into a workaround we could apply?

**Comment by akosyakov** [comment]
Adding it as a dependency? 😬 

I know how to fix in Theia but there are consequences for different adopters since it requires additional system libraries to be installed. I hope we can clarify it on the dev meeting next Tuesday and then someone can send a PR.

**Comment by akosyakov** [comment]
@TimWebb Could you share how you look up keytar? Do you have code like `return r(`${env.appRoot}/node_modules/keytar`);`?

**Comment by nmorenor** [comment]
Hey, on CodeTogether we don't explicitly have the dependency to ```keytar```. And to get it we do something like:

```
function getKeytarModule<T>(): T | undefined {
    const vscodeRequire = eval('require');
    try {
        return vscodeRequire('keytar');
    } catch (err) {
    }
    return undefined;
}
```
For the next release of CodeTogether we have made a change to ignore the error if we can't load ```keytar```. The only side effect will be that at some point you'll need to re-login to be able to start a session.

**Comment by TimWebb** [comment]
We're testing now but basic support has been released as part of 1.1.0.  Things like saving credentials will be coming later.  Also, the Presence view doesn't show right as it is missing the Welcome view.

**Comment by akosyakov** [comment]
@TimWebb Thanks, we will have look at supporting welcomes view too.

Is there a way to join the session from user IDE or one should always use codetogether IDE?
